packages/core/android/src/test/kotlin/com/datadog/reactnative/DdSdkTest.kt

```diff
@@ -1039,6 +1039,70 @@ internal class DdSdkTest {
             }
     }
 
+    // TODO: Fix the next 2 tests to match with interactionTracking
+    @Test
+    fun `𝕄 initialize native SDK 𝕎 initialize() {no user action tracking}`(
```

Comment:
You're missing a test IMHO. You tested what should happen when `DD_NATIVE_INTERACTION_TRACKING` is set to `true` and `false`, but you should add a test for when it's set to `null` as well.